lib/ansible/modules/user.py

```diff
@@ -183,6 +183,15 @@
             - If no passphrase is provided, the SSH key will default to having no passphrase.
         type: str
         version_added: "0.9"
+    ssh_key_rounds:
+        description:
+            - Optionally specify the number of KDF (key derivation function) rounds used to provide higher resistance to brute-force password cracking.
+            - This increases time do decrypt the encrypted private key file.
+            - If this option is not set, it defaults to the OpenSSH default, which is 16.
+            - It only takes effect when O(ssh_key_passphrase) is provided, because in this case the private key is encrypted.
```

Comment:
I have changed this part. But because different distributions and versions have different default values, i still included a sentence to mention this. Is this ok, or should i remove defaults completely?